### ISSUE TITLE
arcgis library installation on macOS fails

### ISSUE THREAD

**Issue by azinsharaf** [Esri/arcgis-python-api#1335]
**Describe the bug**
`arcgis` can't be installed on MacOs (in my case Macbook Pro M1 Max chip).

**To Reproduce**
```terminal
conda create --name gis-enviro
conda install --name gis-enviro -c esri arcgis
```
error:
```python
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - arcgis

Current channels:

  - https://conda.anaconda.org/esri/osx-arm64
  - https://conda.anaconda.org/esri/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

**Platform:**
 - OS: [e.g. iOS] macOS, 12.5

**Comment by azinsharaf** [comment]
#1322 #1323

**Comment by azinsharaf** [comment]
#1322 #1323

**Comment by nanaeaubry** [comment]
We tried to reproduce this on our end but could not reproduce. What version of Python are you using?

**Comment by azinsharaf** [comment]
I'm using 3.9.12. I would remove the anaconda and try it again.
Do you think it would be better to include the compatible python version in the esri document?
so [here](https://developers.arcgis.com/python/guide/understanding-conda/#create-an-environment), instead of `conda create --name gis-enviro`
specify the python version: `conda create --name gis-enviro python=3.9`

**Comment by nanaeaubry** [comment]
Not sure, we have not run into this issue and the person who tried to reproduce said he only gets the same error as you when he inputs a random package name that doesn't exist. So maybe you should upgrade Anaconda and then try again?
https://developers.arcgis.com/python/guide/install-and-set-up/#installation-using-anaconda-for-python-distribution

If you still cannot get it then I would recommend reaching out to tech support: https://support.esri.com/en 
They are more capable of taking a look on a case by case basis

**Comment by azinsharaf** [comment]
got it. Thanks.

**Comment by jtroe** [comment]
We released v2.3.0 of the ArcGIS Python API today.  We are working on adding support for ARM Mac, but have released a prerelease version. Just add the prerelease label when installing:
`conda install -c esri -c esri/label/prerelease arcgis`
We are targeting this being a supported feature in v2.3.1

**Comment by azinsharaf** [comment]
thanks. i will.
